docs/cvl/imports.md

```diff
@@ -5,12 +5,24 @@ Import and Use Statements
 Contents of additional spec files can be imported using the `import` command.
 Some parts of the imported spec files are implicitly included in the importing
 spec file, while others such as rules and invariants must be explicitly
-`use`d.
+`use`d. Functions, definitions, filters and preserved blocks of the imported spec can be overridden by the importing 
+spec.
 
 ```{todo}
 This feature is currently undocumented.
 ```
 
+Examples
+--------
+- [Example for `import`](https://github.com/Certora/Examples/blob/be09cf32c55e39f5f5aa8cba1431f9e519b52365/CVLByExample/import/certora/specs/sub.spec#L1)
+- [`use rule` with filters](https://github.com/Certora/Examples/blob/be09cf32c55e39f5f5aa8cba1431f9e519b52365/CVLByExample/import/certora/specs/sub.spec#L3)
+- [overriding imported filters](https://github.com/Certora/Examples/blob/be09cf32c55e39f5f5aa8cba1431f9e519b52365/CVLByExample/import/certora/specs/sub.spec#L3)
```

Comment:
I would also include a parenthetical link to the imported spec for the more complicated examples

Reply:
what is a " a parenthetical link "?